test/metabase/query_processor/middleware/metrics_test.clj

```diff
@@ -80,6 +81,17 @@
                        :aggregation [[:avg {} [:field {} (meta/id :products :rating)]]]}]}
             (adjust query)))))
 
+(deftest adjust-prometheus-metric-test
```

Comment:
This probably needs more test cases now that we inc the counter in multiple places